### ISSUE TITLE
JAVA_HOME does not set in fish shell

### ISSUE THREAD

**Issue by liaoya** [shyiko/jabba#80]
The following is the output of fish shell:

    tshen@tom-shen ~> jabba ls
    sjre@1.8.131
    tshen@tom-shen ~> jabba use sjre@1.8.131
    tshen@tom-shen ~> echo $JAVA_HOME
    
    tshen@tom-shen ~> which javac
    /home/tshen/.jabba/jdk/sjre@1.8.131/bin/javac

The following is output of bash

    tshen@tom-shen ~> bash
    [tshen@tom-shen ~]$ jabba ls
    sjre@1.8.131
    [tshen@tom-shen ~]$ echo $JAVA_HOME
    
    [tshen@tom-shen ~]$ jabba use sjre@1.8.131
    [tshen@tom-shen ~]$ which javac
    ~/.jabba/jdk/sjre@1.8.131/bin/javac
    [tshen@tom-shen ~]$ echo $JAVA_HOME
    /home/tshen/.jabba/jdk/sjre@1.8.131

It's obvious that jabba does not set JAVA_HOME correctly in fish shell.

**Comment by shyiko** [comment]
@liaoya can you please specify the version of your fish shell? I just tried
```sh
shyiko@shyiko-laptop ~> jabba use 1.9.0-155
shyiko@shyiko-laptop ~> echo $JAVA_HOME 
/home/shyiko/.jabba/jdk/1.9.0-155
shyiko@shyiko-laptop ~> jabba use sjre@1.8.131
shyiko@shyiko-laptop ~> echo $JAVA_HOME 
/home/shyiko/.jabba/jdk/sjre@1.8.131
shyiko@shyiko-laptop ~> fish --version 
fish, version 2.5.0
```

Everything seems to be functioning properly.

P.S. `fish` integration script is located at `~/.jabba/jabba.fish` (it depends on `tr` and `sed`).

**Comment by liaoya** [comment]
@shyiko I use CentOS 7.3 and fish shell 2.5 comes from http://download.opensuse.org/repositories/shells:fish:release:2/CentOS_7

I will try to find more information next Monday.

**Comment by shyiko** [comment]
@liaoya Thank you. I was able to reproduce the behavior you described using `centos:centos7.3.1611` docker image. Fixed in 4631bac888fa25adb0924ffb763594eb52a0047e. Simply [reinstall](https://github.com/shyiko/jabba#installation) jabba for changes to appear. :tada:

**Comment by liaoya** [comment]
@shyiko 
Thank you for the fix. I have verified this by modify ~/.jabba/jabba.fish manually.
